lib/livebook_web/live/session_live.ex

```diff
@@ -664,6 +664,32 @@ defmodule LivebookWeb.SessionLive do
     {:noreply, socket}
   end
 
+  def handle_event("delete_section", %{"section_id" => section_id}, socket) do
+    socket =
+      case Notebook.fetch_section(socket.private.data.notebook, section_id) do
+        {:ok, section} ->
+          if section.cells == [] do
```

Comment:
You could combine `case` and `if` with:

```elixir
{:ok, %{cells: []} = section} ->
  ...
{:ok, section} ->
  ...
```

But either way is great, your call!

Reply:
I thought about doing this right after pushing it up 😂  I will make the change

Reply:
Fixed in https://github.com/livebook-dev/livebook/pull/829/commits/2269ded747fbd1f274b52e0cc04326936e903f3e